theano/gof/opt.py

```diff
@@ -1995,7 +1996,8 @@ def __init__(self,
                  failure_callback=None,
                  ignore_newtrees=True,
                  max_use_ratio=None,
-                 final_optimizers=None):
```

Comment:
Can you quickly document `final_optimizers` and `cleanup_optimizers` in the docstring?